src/functions.jl

```diff
@@ -110,3 +97,33 @@ function Base.append!(A::NamedDimsArray{L,T,1}, B::AbstractVector) where {L,T}
     data = append!(parent(A), unname(B))
     return NamedDimsArray{newL}(data)
 end
+
+################################################
+# Generators
+
+if VERSION > v"1.1-"
```

Comment:
```suggestion
@static if VERSION > v"1.1-"
```

Reply:
```suggestion
@static if VERSION >= v"1.1"
```
No need to worry about prerelease versions